packetbeat/docs/fields.asciidoc

```diff
@@ -626,6 +632,284 @@ The time interval in seconds that this resource record may be cached before it s
 The data describing the resource. The meaning of this data depends on the type and class of the resource record.
 
 
+=== amqp Fields
+
+AMQP specific event fields.
+
+
+==== amqp.reply-code
+
+type: int
+
+example: 404
+
+AMQP reply code to an error, similar to http reply-code
+
+
+==== amqp.reply-text
+
+type: string
+
+Text expliciting the error.
```

Comment:
Can you say "explaining" instead of "expliciting" here? I haven't seen the word "expliciting" in common usage.